diff -urNp -x '*.orig' Audio-1.029.org/Play/Net/Makefile.PL Audio-1.029/Play/Net/Makefile.PL
--- Audio-1.029.org/Play/Net/Makefile.PL	2003-08-05 21:18:56.000000000 +0200
+++ Audio-1.029/Play/Net/Makefile.PL	2021-05-07 23:13:40.859600570 +0200
@@ -18,7 +18,7 @@ if (try_compile("-c ../config/Net.c"))
  {
   WriteMakefile(
       'NAME'     => 'Audio::Play::Net',
-      'LIBS'     => [q[-L/usr/X11R6/lib -L/usr/local/lib -laudio -lX11 -lsocket -lnsl -lm]],
+      'LIBS'     => [q[-L/usr/X11R6/lib -L/usr/local/lib -laudio]],
       'INC'      => q[-I/usr/X11R6/include],
       'clean'    => {FILES => "*% *.bak"},
       'OBJECT'   => '$(O_FILES)',
diff -urNp -x '*.orig' Audio-1.029.org/Play/Net/Net.xs Audio-1.029/Play/Net/Net.xs
--- Audio-1.029.org/Play/Net/Net.xs	2003-08-05 21:18:56.000000000 +0200
+++ Audio-1.029/Play/Net/Net.xs	2021-05-07 23:13:40.859600570 +0200
@@ -67,6 +67,7 @@ AuPlay(AuServer *aud, Audio *au, float v
  int priv = 0;
  AuEvent ev;
  STRLEN samp = Audio_samples(au);
+ STRLEN na;
  Sound s = SoundCreate(SoundFileFormatNone,
                        little_endian ? AuFormatLinearSigned16LSB : AuFormatLinearSigned16MSB,
                        1, au->rate, samp, SvPV_nolen(au->comment)); 
