--- Audio-0.005/Data/Data.pm.orig	Sun Aug  4 21:29:03 2002
+++ Audio-0.005/Data/Data.pm	Sun Aug  4 21:29:21 2002
@@ -1,7 +1,7 @@
 package Audio::Data;
 
 # PerlIO calls used in .xs code
-require 5.00302;  
+require 5.003_02;  
 
 require DynaLoader;
 @ISA = qw(DynaLoader);
--- Audio-0.005/Play/Net/Net.pm.orig	Sun Aug  4 21:38:01 2002
+++ Audio-0.005/Play/Net/Net.pm	Sun Aug  4 21:38:05 2002
@@ -1,6 +1,6 @@
 package Audio::Play::Net;
 # PerlIO calls used in .xs code
-require 5.00302;  
+require 5.003_02;  
 require Audio::Play;
 @ISA = qw(Audio::Play);
 $VERSION = "0.001";
